### DISCUSSION TITLE
How can I override the manager webpack config in storybook v7?

### DISCUSSION THREAD

**Discussion by binomialstew** [storybookjs/storybook#23934] [Help] [answered]
Using Storybook v6, I was able to use `managerWebpack` in main.js, but now that seems to have no effect. Using `webpackFinal` in main.js affects the preview webpack config but does not seem to have any effect on the manager.

**Comment by binomialstew** [comment]
We have custom addons that use some functions and utils from our main project (next.js)--requiring aliasing of imports to avoid use of serverside functionality. We also have certain webpack plugins set up. How can I set these for the storybook manager now that esbuild is used in v7?

**Comment by binomialstew** [comment]
For example, we have a function that uses `createHash` from `crypto`. Without `config.resolve.fallback.crypto = false`, storybook fails to build.

**Comment by hamidrezahanafi** [comment, marked as answer]
Storybook 7 is not using `webpack` for manager but `esbuild` and I couldn't find a way to configure manager bundler.
